app/models/activity.rb

```diff
@@ -356,6 +356,18 @@ def thumbnail_representation
     end
   end
 
+  def google_event
+    return nil if start.nil? || self.end.nil?
+
+    # check locale for correct description
+    description = if I18n.locale == :en
```

Comment:
We (the board) now add the activity URL to the description in the calendar; could you prepend that too?